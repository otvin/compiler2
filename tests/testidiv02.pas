program testidiv02(output);
begin
	writeln('Should be a division by zero error.');
	writeln(30724 div 0);
end.
