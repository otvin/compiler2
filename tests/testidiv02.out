Should be a division by zero error.
Division by zero error